GUI/SearchTaskWindow.xaml

```diff
@@ -420,7 +682,11 @@
                             </GroupBox>
                             <GroupBox Header="Pruned Database Writing">
                                 <StackPanel>
-                                    <CheckBox x:Name="writePrunedDBCheckBox" Content="Write Two Pruned Databases (Mod and Mod+Protein pruned)" />
+                                    <CheckBox x:Name="writePrunedDBCheckBox" Content="Write Two Pruned Databases (Mod and Mod+Protein pruned)">
+                                        <ToolTipService.ToolTip>
+                                            <ToolTip Content="Writes two databases one retains all proteins but only keeps modifications that were found at a 1% FDR while the other removes proteins not found at a 1% FDR." ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000"/>
```

Comment:
This is very uncommon and not recommended for the typical user.

This option writes two databases: one retains all proteins but only keeps modifications that were found at a 1% FDR, while the other removes proteins not found at a 1% FDR. This is intended for further analysis steps, like importing into Proteoform Suite to analyse top-down and intact-mass data.